app/src/Submit/OnlineJudgeFactory/OmegaUp.ts

```diff
@@ -0,0 +1,45 @@
+/*
+    cpbooster "Competitive Programming Booster"
+    Copyright (C) 2020  Sergio G. Sanchez V.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Page } from "playwright-chromium";
+import OnlineJudge, { OnlineJudgeName } from "./OnlineJudge";
+
+export default class Codeforces extends OnlineJudge {
+  readonly onlineJudgeName = OnlineJudgeName.omegaup;
+  readonly loginUrl = "https://omegaup.com/login/";
+  readonly blockedResourcesOnSubmit: Set<string> = new Set(["image", "font"]);
```

Comment:
sometimes we don't need to load "javascript" or "stylesheets" or images etc, by not loading them (blocking them) we can submit files faster. `blockedResourcesOnSubmit` is used to know which resources are not needed. See `Codeforces.ts` and `AtCoder.ts` for more examples.